template/Info.plist

```diff
@@ -14,6 +14,8 @@
 	<string>Public Domain</string>
 	<key>DCSDictionaryManufacturerName</key>
 	<string>Webster</string>
+	<key>DCSDictionaryFrontMatterReferenceID</key>
+	<string>front_back_matter</string>
```

Comment:
What does this do?

Reply:
It lets you go to Go → Front/Back Matter → Webster’s 1913 and see the things that weren’t placed under a word.